static/compare.html

```diff
@@ -1,11 +1,11 @@
 <table><tr>
 <td><iframe id=a src="/?WelcomeVisitors" width=400 height=1000></iframe></td>
-<td><iframe id=b src="http://c2.com/cgi/wikix?WelcomeVisitors" width=400 height=1000></iframe></td>
+<td><iframe id=b src="https://c2.com/cgi/wikix?WelcomeVisitors" width=400 height=1000></iframe></td>
```

Comment:
However it looks like that is a tool you used to compare original to the remodeled version and it is 404ing because the original is no longer being served.

Reply:
Does this `iframe` element need to stay in here if that link is dead?

Reply:
I don’t think the page is linked. Probably worth keeping for debugging if the original wiki renderer is turned on again for debugging.